ui/src/partials/navigation-tree.hbs

```diff
@@ -4,13 +4,13 @@
   <li class="nav-li" data-depth="{{#if ../level}}{{../level}}{{else}}0{{/if}}" {{#if currentPath}}data-state="expanded"
     {{/if}}>
     {{#if url}}
-    <a class="flex shrink align-center link nav-link {{#if (eq url @root.page.url)}}nav-link-active{{/if}}"
+    <a class="shrink align-center link nav-link {{#if (eq url @root.page.url)}}nav-link-active{{/if}}"
```

Comment:
I think `shrink` and `align-center` are only relevant if `flex` is used, but can you please double check?

```suggestion
    <a class="link nav-link {{#if (eq url @root.page.url)}}nav-link-active{{/if}}"
```